packages/server/src/api/controllers/rowAction/crud.ts

```diff
@@ -68,26 +69,6 @@ export async function create(
   ctx.status = 201
 }
 
-export async function update(
```

Comment:
This logic has not been moved to the automation screen, so this is not needed anymore